ui/apps/platform/cypress/integration/integrations/notifiers.test.js

```diff
@@ -539,23 +539,14 @@ describe('Notifier Integrations', () => {
             getInputByLabel('Annotation key for Slack webhook').click().blur();
 
             getHelperElementByLabel('Integration name').contains('Name is required');
-            getHelperElementByLabel('Default Slack webhook').contains('Slack webhook is required');
-            cy.get(selectors.buttons.test).should('be.disabled');
-            cy.get(selectors.buttons.save).should('be.disabled');
-
-            // Step 2, check fields for invalid formats
-            getInputByLabel('Integration name').clear().type(integrationName);
-            getInputByLabel('Default Slack webhook')
-                .clear()
-                .type('https://hooks.slack.com/services/')
-                .blur();
```

Comment:
It looks like these two fields need to have values filled in for the test to complete successfully even if we are not checking for invalid formats. IMO this should be done after the two `should('be.disabled')` checks for the test/save buttons since testing that is still valid.

Reply:
Yeah, I selected too much to delete. Fixing that now.